spec/controllers/idv/socure/document_capture_controller_spec.rb

```diff
@@ -269,6 +269,37 @@
         expect(response).to redirect_to(idv_unavailable_path)
       end
     end
+
+    context 'reuse of valid capture app urls when appropriate' do
+      let(:fake_capture_app_url) { 'https://verify.socure.test/fake_capture_app' }
+      let(:socure_capture_app_url) { 'https://verify.socure.test/' }
+      let(:docv_transaction_token) { '176dnc45d-2e34-46f3-82217-6f540ae90673' }
+      let(:response_body) do
+        {
+          referenceId: '123ab45d-2e34-46f3-8d17-6f540ae90303',
+          data: {
+            eventId: 'zoYgIxEZUbXBoocYAnbb5DrT',
+            docvTransactionToken: docv_transaction_token,
+            qrCode: 'data:image/png;base64,iVBO......K5CYII=',
+            url: socure_capture_app_url,
+          },
+        }
+      end
+
+      before do
+        allow(request_class).to receive(:new).and_call_original
+        allow(I18n).to receive(:locale).and_return(expected_language)
+        allow(DocumentCaptureSession).to receive(:find_by).and_return(document_capture_session)
```

Comment:
this is an ActiveRecord model, is there a way to create a real instance in the DB instead of stubbing the whole model class?

Reply:
I'm not sure what you mean here. In line 292 I am referencing a created test record, line 290 is needed to ensure the class is not being called (Line 299), and I believe Line 291 was put to resolve a similar error I was having.

Reply:
I'm saying that I think that `allow(DocumentCaptureSession).to receive(...)` seems more delicate to me than doing

```
create(:document_capture_session, uuid: ....)
```

to create a real record instead of stubbing.

And re: not creating a new record, we could also assert that the number of rows does not change ( `DocumentCaptureSession.count`)

Reply:
I agree with @zachmargolis here ... can we refactor this?

Reply:
i've created a [ticket](https://cm-jira.usa.gov/browse/LG-15246) to address this use of DocumentCaptureSession which pervades throughout the document capture specs